app/controllers/features_controller.rb

```diff
@@ -12,14 +12,14 @@ def embed
   end
 
   def new
-    if current_user.role != 'admin'
+    unless current_user.admin?
```

Comment:
hi @seafr , in all other places, you have used `if !current_user.admin?`. how about following the same syntax here too, only if you want to?

Reply:
I used `if !` elsewhere due to the presence of the `else` clause. I find it hard to understand `else` after `unless`, else I'd use `unless` everywhere.

P.S. I have a feeling there is an overuse of *else* in the above text though.

Reply:
yeah `unless/else` trips me up too. Interestingly -- I think double negatives also vary in use across languages!